src/_layouts/base.njk

```diff
@@ -22,14 +22,14 @@ title: CS++
 
     <script defer data-domain="cspp.ie" src="https://plausible.io/js/script.js"></script>
 
-    <script src="https://cdn.jsdelivr.net/npm/ua-parser-js@1.0.37/src/ua-parser.min.js"></script>
+    <script async src="https://cdn.jsdelivr.net/npm/ua-parser-js@1.0.37/src/ua-parser.min.js"></script>
```

Comment:
The `integrity` attribute lets you validate hashes when importing. It could be a useful feature here: https://www.w3schools.com/tags/att_script_integrity.asp

Reply:
No integrity provided with UA parser.
Not entirely arsed to generate a checksum